### ISSUE TITLE
[Bug] variables sorting does not consider numbers

### ISSUE THREAD

**Issue by massimo-cassandro** [willofindie/vscode-cssvar#120]
**Describe the bug**
Variables are sorted alphabetically and numbers suffix or prefix are not in the right sequence

<img width="164" alt="Screenshot 2024-09-24 alle 12 11 03" src="https://github.com/user-attachments/assets/24a71927-dc1d-4c11-ad97-f4a1dc8821be">

**Expected behavior**
Variable must be sorted in natural order:

```
--var-1
--var-2
...
--var-10
--var-11
```

**Details (please complete the following information):**
 - OS: OSX Ventura 13.7
 - VSCode version 1.93.1
 - Extension version 2.6.4

**Comment by phoenisx** [comment]
Try using [cssvar.disableSort](https://github.com/willofindie/vscode-cssvar/blob/main/feature-contributions.md) extension setting.

**Comment by phoenisx** [comment]
![Screenshot from 2024-09-24 22-50-48](https://github.com/user-attachments/assets/e36fbb52-33b1-4a36-b130-308747df64ca)

It's working fine for me.
**_Did u restart/reload your VSCode Window after changing the extension settings?_**

**Comment by massimo-cassandro** [comment]
You are right: after reloading my vscode window, the extension seems to work as expected. 

But this leads to another problem: I usually load custom properties from more than one file and when I disable sorting, they get merged in an unpredictable way:

<img width="286" alt="Screenshot_2024-09-25_alle_14_18_35" src="https://github.com/user-attachments/assets/3232eea2-293a-4adb-a5cb-a85fb1a8b66f">

**Comment by phoenisx** [comment]
Agreed! That needs to be fixed.
I have low bandwidth.
I'll keep this issue open to work on this later.

_If you are interested, care to open a PR?_

**Comment by massimo-cassandro** [comment]
> If you are interested, care to open a PR?

I'd like to, but I dont know if I will have time.
Anyway I'll give it a look

Thanks
